packages/paste-core/components/tabs/__tests__/tabs.test.tsx

```diff
@@ -1,11 +1,39 @@
 import * as React from 'react';
+import {matchers} from 'jest-emotion';
 import {render, screen} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore typescript doesn't like js imports
 import axe from '../../../../../.jest/axe-helper';
 import {HorizontalTabs, StateHookTabs} from '../stories/index.stories';
 import {Tabs, Tab, TabList, TabPanels, TabPanel} from '../src';
+import {getElementName} from '../src/utils';
+
+expect.extend(matchers);
 
 describe('Tabs', () => {
+  describe('Utils', () => {
+    describe('getElementName', () => {
+      const mockFallbackName = 'TEST_ELEMENT_FALLBACK';
+
+      it('should return the correct string when element is undefined', () => {
+        expect(getElementName('horizontal', mockFallbackName)).toEqual('HORIZONTAL_TEST_ELEMENT_FALLBACK');
+        expect(getElementName('vertical', mockFallbackName)).toEqual('VERTICAL_TEST_ELEMENT_FALLBACK');
+      });
+      it('should return the correct string when element is null', () => {
+        expect(getElementName('horizontal', mockFallbackName, null)).toEqual('HORIZONTAL_TEST_ELEMENT_FALLBACK');
+        expect(getElementName('vertical', mockFallbackName, null)).toEqual('VERTICAL_TEST_ELEMENT_FALLBACK');
+      });
+      it('should return the correct string when element is defined', () => {
+        expect(getElementName('horizontal', mockFallbackName, 'CUSTOM_NAME')).toEqual('CUSTOM_NAME');
+        expect(getElementName('vertical', mockFallbackName, 'CUSTOM_NAME')).toEqual('CUSTOM_NAME');
+      });
+      it('should return the correct string when element is an empty string', () => {
```

Comment:
Why do we want it to return an empty string when that's passed in? What would be the use case for that?

Reply:
The `element` is typed to be a `string` which can include empty strings. I can't think of great example for a use case off the top of my head, but I think returning an empty string, if that is provided as the `element` prop is appropriate given the typing.

This is the case as well for the other components that have been updated for customization. 
In those instances, we are using default values assigned where the props are destructured. For example:

```
const Component = ({ element =  'EXAMPLE', children }) => (<Box element={element}>{children}</Box>)
```

Note that the default value is only used if element is implicitly undefined (e.g. the prop `element` is not set) or explicitly undefined (e.g. `element: undefined`). Even though both undefined and `''` are technically falsy values, if we pass in an empty string as the value of `element`, the data attribute will be set to an empty string.

I think this could be improved with development-only console warning logs might be useful, to help folks catch this if it is done unintentionally.